Content.Tests/DMTests.cs

```diff
@@ -114,24 +111,25 @@ public void OneTimeSetup()
         private (bool Success, DreamValue? Returned, Exception? except) RunTest() {
             var prev = _dreamMan.LastDMException;
 
-            DreamValue? result = null;
+            DreamValue? retValue = null;
             Task<DreamValue> callTask = null;
 
             DreamThread.Run("RunTest", async (state) => {
                 if (_objectTree.TryGetGlobalProc("RunTest", out DreamProc proc)) {
                     callTask = state.Call(proc, null, null, new DreamProcArguments(null));
-                    result = await callTask;
+                    retValue = await callTask;
                     return DreamValue.Null;
                 } else {
                     Assert.Fail($"No global proc named RunTest");
                     return DreamValue.Null;
                 }
             });
 
+
             var Watch = new Stopwatch();
             Watch.Start();
 
-            // Tick until our inner call has finished
+             // Tick until our inner call has finished
```

Comment:
Unnecessary space created
```suggestion
            // Tick until our inner call has finished
```